### ISSUE TITLE
Meteor 1.5: Error: No metadata files found for isopack immediately following meteor create . --full

### ISSUE THREAD

**Issue by joshuadavidson** [meteor/meteor#8878]
Hey there. I was spinning up a new meteor 1.5 app today on Win10 64bit and decided to use the `meteor create . --full` command to create an app. Immediately after running the create command I run `meteor run` and get the error below. Seems like quite a bug if the scaffold app immediately throws the error--I'm surprised no one has reported this yet. It makes me wonder if there is something up with my machine. Can anyone reproduce?

I ran out of time today but tomorrow I'll try running `meteor create . --base` and just `meteor create` and report back. Hopefully this will help narrow down the issue.

```
C:\Users\Joshua Davidson\Documents\Coding\rai-petroai>meteor run
[[[[[ C:\Users\Joshua Davidson\Documents\Coding\rai-petroai ]]]]]

=> Started proxy.
=> Started MongoDB.
C:\Users\Joshua Davidson\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:218
      throw error;
      ^

Error: No metadata files found for isopack at: /C/Users/Joshua Davidson/AppData/Local/.meteor/packages/coffeescript/1.0.17
    at [object Object]._.extend._loadUnibuildsFromPath (C:\tools\isobuild\isopack.js:901:13)
    at C:\tools\packaging\tropohouse.js:521:21
    at Array.forEach (native)
    at Function._.each._.forEach (C:\Users\Joshua Davidson\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:79:11)
    at C:\tools\packaging\tropohouse.js:520:13
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at C:\tools\packaging\tropohouse.js:515:22
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at Object.download (C:\tools\packaging\tropohouse.js:427:20)
    at C:\tools\packaging\tropohouse.js:613:18
    at C:\tools\utils\buildmessage.js:572:21
    at C:\Users\Joshua Davidson\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\fiber_pool.js:32:39
 => awaited here:
    at Promise.await (C:\Users\Joshua Davidson\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:60:12)
    at Object.forkJoin (C:\tools\utils\buildmessage.js:604:6)
    at [object Object]._.extend.downloadPackagesMissingFromMap (C:\tools\packaging\tropohouse.js:609:18)
    at C:\tools\project-context.js:808:25
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at C:\tools\project-context.js:807:20
    at Object.catalog.runAndRetryWithRefreshIfHelpful (C:\tools\packaging\catalog\catalog.js:141:3)
    at ProjectContext._downloadMissingPackages (C:\tools\project-context.js:806:13)
    at C:\tools\project-context.js:283:9
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at ProjectContext._.extend._completeStagesThrough (C:\tools\project-context.js:273:18)
    at C:\tools\project-context.js:265:12
    at Function.run (C:\tools\tool-env\profile.js:490:12)
    at ProjectContext._.extend.prepareProjectForBuild (C:\tools\project-context.js:264:13)
    at C:\tools\runners\run-app.js:563:29
    at C:\tools\utils\buildmessage.js:271:13
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at bundleApp (C:\tools\runners\run-app.js:562:31)
    at [object Object]._.extend._runOnce (C:\tools\runners\run-app.js:631:35)
    at [object Object]._.extend._fiber (C:\tools\runners\run-app.js:890:28)
    at C:\tools\runners\run-app.js:417:12
```

**Comment by abernix** [comment]
I'm assuming this problem is related to the space in your username (`Joshua Davidson`).  See https://github.com/meteor/meteor/issues/8840.  If you can confirm (perhaps create a new user and see if the problem goes away?) then this should be closed in favor of that existing issue.

**Comment by joshuadavidson** [comment]
@abernix You were correct. I created a local user without spaces and everything worked perfectly. Sorry for the duplicated issue feel free to close.

**Comment by abernix** [comment]
Glad to hear that's the problem! We'll get this fixed via #8840.  Thanks for reporting back and testing!
